src/main/resources/i18n/messages.properties

```diff
@@ -1,4 +1,18 @@
 Api.Error.ApiKeyNotAvail=Weak or no server API key configured. UMS.conf api_key should have at least 12 characters.
+Api.Error.UnknownService=No service available for path
+Api.Playlist.AudiotrackIdUnknown=Unknown audiotrack id
+Api.Playlist.PlaylistAlreadyExists=Playlist already exists
+Api.Playlist.PlaylistCanNotBeCreated=Playlist can not be created
+Api.Playlist.PlaylistCreated=Playlist has been created
+Api.Playlist.PlaylistNameNoExt=Provided playlist name shall have no path and no extension
```

Comment:
```suggestion
Api.Playlist.PlaylistNameNoExt=Provided playlist name has no path or file extension
```

Reply:
ok, the intension is to provide neither a path nor a file extension. I'll give it a new try ...